MAUI/AIAssistView/working-with-aiassistview.md

```diff
@@ -0,0 +1,279 @@
+platform: MAUI
+control: SfAIAssistView
+documentation: ug
+---
+
+# Working with AI AssistView in .NET MAUI AI AssistView (SfAIAssistView)
+
+## Stop Responding in .NET MAUI AI AssistView (SfAIAssistView)
+
+The [SfAIAssistView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.AIAssistView.SfAIAssistView.html) control provides `Stop Responding` feature that allows you to cancel an ongoing AI response by clicking the Stop Responding button. This feature ensures that users can stop if a response is no longer needed.
+By default, the Stop Responding button is displayed, to disable this set the `EnableStopResponding` property to `false`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfAIAssistView x:Name="sfAIAssistView"
+                               EnableStopResponding="False"/>  
+
+{% endhighlight %} 
+{% highlight c# hl_lines="6" %}
+
+    SfAIAssistView sfAIAssistView; 
+    public MainPage() 
+    { 
+        InitializeComponent(); 
+        this.sfAIAssistView = new SfAIAssistView();
+        this.sfAIAssistView.EnableStopResponding = false;
+        this.Content = sfAIAssistView; 
+     } 
+
+{% endhighlight %}
+{% endtabs %}
+
+### Stop Responding Event and Command
+
+The `SfAIAssistView` control includes a built-in event called `StopResponding` and a command named `StopRespondingCommand`. These are triggered when the `Stop Responding` button is clicked.
+To cancel the response using the `StopRespondingCommand` or `StopResponding` event, you can include logic to stop the ongoing response as shown below. 
+
+#### StopResponding Event
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfAIAssistView x:Name="sfAIAssistView"
+                               StopResponding="SfAIAssistView_StopResponding" />
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+
+    sfAIAssistView.StopResponding += SfAIAssistView_StopResponding;
+
+    private void SfAIAssistView_StopResponding(object sender, EventArgs e)
+    {
+       // Handle the Stop Responding action
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### StopResponding Command
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfAIAssistView x:Name="sfAIAssistView"  
+                               StopRespondingCommand="{Binding StopRespondingCommand}" />
+
+{% endhighlight %}
+{% highlight c# tabtitle="ViewModel.cs" hl_lines="10" %}
+
+    public class ViewModel : INotifyPropertyChanged
+    {
+        public ICommand StopRespondingCommand { get; set; }
+
+        public ViewModel()
+        {
+            StopRespondingCommand = new Command(ExecuteStopResponding);
+        }
+
+        private void ExecuteStopResponding()
+        {
+            // logic to handle the Stop Responding action
+            this.CancelResponse = true;
```

Comment:
Include `you canceled the response` response item code snippets.

Reply:
Done as per request